src/platform_impl/apple/appkit/view.rs

```diff
@@ -828,19 +828,19 @@ impl WinitView {
         this
     }
 
-    fn window(&self) -> Retained<WinitWindow> {
+    fn window(&self) -> Retained<NSWindow> {
         let window = (**self).window().expect("view must be installed in a window");
 
-        if !window.isKindOfClass(WinitWindow::class()) {
-            unreachable!("view installed in non-WinitWindow");
+        if !window.isKindOfClass(NSWindow::class()) {
```

Comment:
Nit: This check and the cast below is now redundant, `NSView::window` always returns `NSWindow`.